testsuites/basic/lsvmbus.py

```diff
@@ -0,0 +1,94 @@
+import math
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Windows
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lscpu, Lsvmbus
+
+
+class VmbusNames:
+    def __init__(self, is_gen1: bool) -> None:
+        self.names = [
+            "Operating system shutdown",
+            "Time Synchronization",
+            "Heartbeat",
+            "Data Exchange",
+            "Synthetic mouse",
+            "Synthetic keyboard",
+            "Synthetic network adapter",
+            "Synthetic SCSI Controller",
+        ]
+        if is_gen1:
+            self.names.append("Synthetic IDE Controller")
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite uses to check correctness of vmbus channel
+    """,
+    tags=[],
+)
+class LsVmBus(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test is to check below vmbus names exist
+            - Operating system shutdown
+            - Time Synchronization
+            - Heartbeat
+            - Data Exchange
+            - Synthetic mouse
+            - Synthetic keyboard
+            - Synthetic network adapter
+            - Synthetic SCSI Controller
+            - Synthetic IDE Controller (gen1 only)
+        Check channel counts of each netvsc and storvsc SCSI device.
```

Comment:
use a bullet points for different test points. Current format is hard to read, and no structure.